libyul/AsmAnalysis.h

```diff
@@ -78,6 +78,11 @@ class AsmAnalyzer
 	/// Performs analysis on the outermost code of the given object and returns the analysis info.
 	/// Asserts on failure.
 	static AsmAnalysisInfo analyzeStrictAssertCorrect(Dialect const& _dialect, Object const& _object);
+	static AsmAnalysisInfo analyzeStrictAssertCorrect(
+		Dialect const& _dialect,
+		Block const& _block,
```

Comment:
```suggestion
		Block const& _astRoot,
```

That's the general naming I would use for it in other places too, unless the function can actually work with any random block, not just the top-level one.